sagemaker_processing/fairness_and_explainability/text_explainability_sagemaker_algorithm/byo_blazingtext_model_hosting.ipynb

```diff
@@ -0,0 +1,892 @@
+{
```

Comment:
<p>Can you provide an estimate of the training time using the default options in the notebook? 10 minutes, 30 minutes, an hour, ...? </p><p> </p><p>Formatting nit: extra space: <code>only  remaining</code> -&gt; <code>only remaining</code> ﻿</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Fixed